views/pages/guides/runtime-environments.hbs

```diff
@@ -99,7 +99,7 @@ if (global.Intl) {
     if (!areIntlLocalesSupported(localesMyAppSupports)) {
         // `Intl` exists, but it doesn't have the data we need, so load the
         // polyfill and replace the constructors with need with the polyfill's.
-        require('intl');
+        var IntlPolyfill = require('intl');
```

Comment:
this is not an error, this is just to guarantee that `intl` module executes before the rest of this code.